### PR TITLE
update finance with facebooks

### PR DESCRIPTION
**Pull request by stephengroat**

### CODE REVIEW

File: _data/finance.yml [hunk 1]
```diff
@@ -193,6 +206,7 @@ websites:
     - name: Shoeboxed
       url: https://www.shoeboxed.com/
       twitter: Shoeboxed
+      facebook: shoeboxed
```
Comment (Jawshy) [inline _data/finance.yml:209]: I received this message - including when logged in - when trying to view their Facebook page: 

```
Sorry, this content isn't available at the moment
The link you followed may have expired, or the Page may only be visible to an audience that you aren't in.
Go back to the previous page · Go to News Feed · Visit our Help Centre
```

Is anyone else experiencing this?

File: _data/finance.yml [hunk 2]
```diff
@@ -208,6 +222,7 @@ websites:
       img: transferwise.png
       tfa: No
       twitter: TransferWise
+      facebook: transferwiseUS
```
Comment (Jawshy) [inline _data/finance.yml]: As with https://github.com/2factorauth/twofactorauth/pull/2164, what about using non-region specific pages instead? ([transferwise](https://www.facebook.com/transferwise/))

File: _data/finance.yml (hunk 1 above)
Comment (stephengroat) [reply _data/finance.yml:209]: i think this, like the `spotify` etc. stuff, is a result of facebook serving pages based off of some sort of geographic information (IP address, facebook profile, etc)

File: _data/finance.yml [hunk 3]
```diff
@@ -49,14 +54,16 @@ websites:
...
+      url: http://www.hellowallet.com
       twitter: HelloWallet
+      facebook: hellowallet
       img: hellowallet.png
       tfa: No
 
     - name: InEx Finance
       url: https://www.inexfinance.com/
       twitter: InexFinance
+      facebook: inex.finance
```
Comment (Jawshy) [inline _data/finance.yml:66]: I'm not sure about this one. I can't find a link to it from their website. I only end up at [fentury](https://www.facebook.com/fentury).

**Review by Jawshy — CHANGES_REQUESTED** [review]
Everything looks good except this one teensy-tiny lil' thing.

File: _data/finance.yml (hunk 3 above)
Comment (stephengroat) [reply _data/finance.yml:66]: maybe geo-specific
screenshot 2016-12-11 at 14 53 44

**Review by Jawshy — APPROVED** [review]

File: _data/finance.yml [hunk 4]
```diff
@@ -49,14 +54,16 @@ websites:
       doc: https://support.gusto.com/hc/en-us/articles/210838787-Two-Step-Authentication
 
     - name: HelloWallet
-      url: https://www.hellowallet.com
+      url: http://www.hellowallet.com
```
Comment (mxxcon) [inline _data/finance.yml]: Maybe leave it as https in case in the future they won't downgrade? And similar mindset for other similar changes?

Comment (Jawshy) [reply _data/finance.yml]: Done - https://github.com/2factorauth/twofactorauth/pull/2165/commits/4764ece46eb1ee8946668475a7af42dbd7c03f38.

**Pull request merged**